README.md

```diff
@@ -78,10 +78,10 @@ Quick Guide
 ```javascript
 var Queue = require('bull');
 
-var videoQueue = Queue('video transcoding', 6379, '127.0.0.1');
-var audioQueue = Queue('audio transcoding', 6379, '127.0.0.1');
-var imageQueue = Queue('image transcoding', 6379, '127.0.0.1');
-var pdfQueue = Queue('pdf transcoding', 6379, '127.0.0.1');
+var videoQueue = Queue('video transcoding', {redis: {port: 6379, host: '127.0.0.1'}});
```

Comment:
Might be clearer to use the redis string version here, eg `redis://127.0.0.1:6379`, since the documentation lists the `redisConnectionString` as the second option (albeit optional - but some people might not notice that)